### DISCUSSION TITLE
Button number format

### DISCUSSION THREAD

**Discussion by Raiden38** [bitfocus/companion#2850] [Q&A] [answered]
Hi! A while ago, seems that the buttons numbers format has been changed. Before, button was page.button such as 4.07 but now it seems like it's 4/0/6. I find it more complicated to add to to streamdeck as the numbers does not match (4/0/6 in companion but 4.07 in streamdeck). Is there a way to put companion back to a page.XX format? Thanks!

**Comment by Julusian** [comment, marked as answer]
The plugin for the elgato software will be updated at some point to match this format. Unfortunately until that happens then you are required to figure out the translation yourself, which is not ideal.
You can follow the issue https://github.com/bitfocus/io.bitfocus.companion-plugin/issues/8 for updates on when this gets done.

The format was changed as it is now possible to make the grid larger and smaller. Doing that with the old system would mean that the meaning of `1.12` would be different depending on what the current grid size was

**Reply by Raiden38** [reply]
Awesome, thanks so much for the quick reply, can't wait to get the update on the plugin. Thanks!
